LEGO1/elevatorbottom.cpp

```diff
@@ -0,0 +1,52 @@
+#include "elevatorbottom.h"
+
+// 0x100f04ac
+static char* g_elevatorBottomClassName = "ElevatorBottom";
+
+// OFFSET: LEGO1 0x10017e90 STUB
+ElevatorBottom::ElevatorBottom()
+{
+  // TODO
+}
+
+// OFFSET: LEGO1 0x10018060 STUB
+ElevatorBottom::~ElevatorBottom()
+{
+  // TODO
+}
+
+// OFFSET: LEGO1 0x10018150 STUB
+long ElevatorBottom::Notify(MxParam &p)
+{
+  // TODO
+
+  return 0;
+}
+
+// OFFSET: LEGO1 0x10017f20 STUB
+const char *ElevatorBottom::GetClassName() const
+{
+  return g_elevatorBottomClassName;
+}
+
+// OFFSET: LEGO1 0x10017f30 STUB
+MxBool ElevatorBottom::IsClass(const char *name) const
+{
+  // TODO
+
+  return MxBool();
+}
+
+// OFFSET: LEGO1 0x10017f10 STUB
+undefined4 ElevatorBottom::VTable0x5c()
```

Comment:
Return type is 1 byte
```suggestion
undefined ElevatorBottom::VTable0x5c()
```